.github/workflows/release.yml

```diff
@@ -88,21 +88,21 @@ jobs:
             - name: Push to npm
               run: |
                   yarn npm-publish
-              env:
-                  DEVCYCLE_PROD_SLEUTH_API_TOKEN: ${{ secrets.SLEUTH_API_KEY }}
               if: inputs.draft != true
 
             - name: Push to npm (dry run)
               run: |
                   yarn npm-publish --dry-run
-              env:
-                  DEVCYCLE_PROD_SLEUTH_API_TOKEN: ${{ secrets.SLEUTH_API_KEY }}
               if: inputs.draft == true
 
             - name: Build UMD bundles
               run: |
                   yarn nx build:cdn js
 
             - name: Upload CDN Release
-              run: aws s3 sync dist/sdk/js s3://js.devcycle.com/ --include "devcycle.min.js*" --acl public-read
+              run: |
+                JQ_PATH=".version"
+                SDK_VERSION="$(cat package.json | jq -r $JQ_PATH)"
```

Comment:
what package.json is being read here?

Reply:
ah.. right now this will read the root `package.json` file, which isn't the right one... let me update to point to the `sdk/js/package.json` file as that will be the right SDK version